bin/cylc-check-software

```diff
@@ -49,11 +49,13 @@ opt_spec = {
     'framed': [None, 'LATEXGUIDE', 'TEX'],
     'preprint': [None, 'LATEXGUIDE', 'TEX'],
     'tex4ht': [None, 'LATEXGUIDE', 'TEX'],
-    'TeX': [(3, 0), 'LATEXGUIDE', 'OTHER', ('tex', '-v', r'TeX ([^\s]+)')],
+    'TeX': [(3, 0), 'LATEXGUIDE', 'OTHER',
+            (['tex'], '-v', r'TeX ([^\s]+)')],
     'graphviz': [None, 'GUIORGRAPH', 'OTHER',
-                 ('dot', '-V', r'graphviz version ([^\s]+)', 2)],
-    'imagemagick': [None, 'HTMLUGUIDE', 'OTHER',
-                    ('magick identify', '-version', r'ImageMagick ([^\s]+)')]
+                 (['dot'], '-V', r'graphviz version ([^\s]+)', 2)],
+    'ImageMagick': [None, 'HTMLUGUIDE', 'OTHER',
+                    (['magick identify', 'identify', 'convert'],
+                     '-version', r'ImageMagick ([^\s]+)')]
```

Comment:
It's probably not necessary to check both `identify` and `convert` as they come together in the one package.  On the other hand, I guess there's no harm in having both!

Reply:
If `[magick] identify` succeeds in providing stdout from which a version can be extracted the checking of further commands gets skipped, so yes there is no harm, but will remove it for thoroughness. It's not entirely clear from the official site & docs which base command is the standard for version checking, though I get the impression `convert` is the more commonly used.

Reply:
Agreed. Looks like `convert` is the one to use.